rules/linux/defense_evasion_attempt_to_disable_iptables_or_firewall.toml

```diff
@@ -17,10 +17,40 @@ index = ["logs-endpoint.events.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Attempt to Disable IPTables or Firewall"
+note = """## Triage and analysis
+
+### Investigating Attempt to Disable IPTables or Firewall
+
+Iptables is a firewall program for Linux. It will monitor traffic from and to your server using tables.
+These tables contain sets of rules, called chains, that will filter incoming and outgoing data packets.
+The rule identifies executions to disable iptables or firewalld in an attempt to affect how a host is allowed to
+receive or send network traffic.
+
+#### Possible investigation steps
+
+- Identify the session entry leader and session user.
+- Examine the contents of session leading to the iptables or firewalld disable exectuions via the Session View.
+- Examine the command execution pattern within the active session, which may lead to discovering suspicious activities.
+- Examine processes or services started after the iptables or firewalld service has been disabled, which could indicate malicious activity.
+- Investigate other alerts associated with the user/host during the past 48 hours.
+
+### False positive analysis
+
+- Infrastructure deployment or operation scripts trying to stop iptables or firewalld services.
```

Comment:
I think we normally explain why certain events or activity might create false positives, right @w0rk3r? If not, feel free to ignore this comment.